server/requirements.txt

```diff
@@ -2,5 +2,6 @@ gunicorn==19.6.0
 honcho==0.6.6
 pyodbc==3.0.10
 unidecode==0.04.19
+celery==3.1.25
```

Comment:
I think it should be `celery[redis]==3.1.25` otherwise it will be missing redis. we had a broken build after this change